.github/workflows/witness.yml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 The Archivista Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+on:
+    workflow_call:
+        inputs:
+            pull_request: 
+                required: true
+                type: boolean
+            command:
+                required: true
+                type: string
+            step:
+                required: true
+                type: string
+            attestations:
+                required: true
+                type: string
+            
+jobs:
+    witness:
+        runs-on: ubuntu-latest
+        steps:
+          - uses: actions/checkout@v4.1.1
+          - uses: actions/setup-go@v4.1.0
+            with:
+              go-version: 1.21.x
+
+          - if: ${{ inputs.pull_request == false }}
+            uses: testifysec/witness-run-action@40aa4ef36fc431a37de7c3faebcb66513c03b934
+            with:
+              step: ${{ inputs.step }}
+              attestations: $${ inputs.attestations }}
+              command: /bin/sh -c "$${ inputs.command }}"
```

Comment:
Can I ask, does the `$${ ... }}` have special meaning here? A little thrown off by the imbalanced curly braces and double `$$` as I'm unfamiliar with any special context they have in GitHub workflows and I was unable to find any documentation after a thorough search. I know GitLab uses it to escape another `$` for when auto variable expansion is enabled. Also that shell itself uses it as a PID reference. 

It seems to be getting interpolated as expected by the action so I assume either there's some special meaning, or its auto escaping these special characters through some means. Or maybe I'm wildly missing something... 😅

Reply:
Nope, this was just wrong. 😅 And I deleted the branch too fast to fix it here.  😞

Reply:
https://github.com/testifysec/archivista/pull/79